.github/workflows/build.yml

```diff
@@ -63,13 +63,14 @@ jobs:
     - name: Bootstrap
       run: |
         sudo apt-get update
-        sudo apt-get --no-install-recommends install -y clang-format-14 clang-tidy-14 shellcheck
+        sudo apt-get --no-install-recommends install -y clang-format-14 clang-tidy-14 shellcheck iwyu
```

Comment:
Also, it would be great if we could update the `bootstrap` script to install `iwyu`. 


Would be helpful to have it in `install_packages_brew` (for developers using macOS ) in addition to `install_packages_apt`

Reply:
SGTM. And I will update the bootstrap script.